addons/mail/static/src/core/web/chatter.js

```diff
@@ -379,7 +388,8 @@ export class Chatter extends Component {
         this.closeSearch();
         const schedule = async (threadId) => {
             await this.activityService.schedule(this.props.threadModel, [threadId]);
-            this.load(this.props.threadId, ["activities", "messages"]);
+            const thread = this.store.Thread.get({ model: this.props.threadModel, id: threadId });
```

Comment:
You need to get the thread before the async method (the call to `schedule`), otherwise threadModel might be different.